src/styles/nav.scss

```
@@ -0,0 +1,42 @@
+@import './header';
+
+.nav__link {
```

```suggestion
.nav {
  &__link {
}
```
you can use nesting even in such cases